console_backend/src/process_messages.rs

```diff
@@ -127,9 +127,9 @@ pub fn process_messages(
         }
     })
     .expect(PROCESS_MESSAGES_FAILURE);
-
     let err = messages.take_err();
     let handle = messages.into_handle();
+    handle.thread().unpark();
```

Comment:
I feel like this is kind of a weird place to put the unpark, to me it made more sense after the stopToken is set (e.g. see the example in the rust std: https://doc.rust-lang.org/std/thread/fn.park.html#examples

However, I think this should be fine as long as the parking timeout we are awaking here isn't used to rate limit the `process_messages` function.